dev-lang/tk/tk-8.6.10.recipe

```diff
@@ -93,10 +110,25 @@ INSTALL()
 	mv $libDir/pkgconfig/sdl2tk.pc $libDir/pkgconfig/tk.pc
 
 	# add standard symlinks
-	ln -s $binDir/wish8.6 $binDir/wish
+	ln -s $commandBinDir/wish8.6 $commandBinDir/wish
 
-	prepareInstalledDevelLibs libtkstub8.6
+	prepareInstalledDevelLib libtkstub8.6
 	fixPkgconfig
+
+	# since we pretend to be regular tk
```

Comment:
So far this works for building the _tkinter module for python3.10, haven't had any luck so far enabling this for prior python versions